src/NLog.StructuredLogging.Json.Tests/LoggingException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace NLog.StructuredLogging.Json.Tests
+{
+    [Serializable]
```

Comment:
I have removed the `[Serializable]` attribute, It's not needed and compiles but does nothing in .net core.